sfa_dash/blueprints/base.py

```diff
@@ -198,6 +234,19 @@ def flash_api_errors(self, errors):
         for error in to_flash:
             flash(error, 'error')
 
+    def safe_metadata(self):
+        """Creates a copy of the metadata attribute without the
+        `extra_parameters` key.
+        """
+        safe_metadata = deepcopy(self.metadata)
+        safe_metadata.pop('extra_parameters', None)
+        if 'site' in safe_metadata:
+            safe_metadata['site'].pop('extra_parameters', None)
+        if 'aggregate' in safe_metadata:
+            safe_metadata['aggregate'].pop('extra_parameters', None)
```

Comment:
hmm would it be safer to recurse through and pop any 'extra_parameters' keys found?